### ISSUE TITLE
how to Import private key "wallet.aes.json" file from blockchain.info?

### ISSUE THREAD

**Issue by Soidea** [Multibit-Legacy/multibit-hd#992]
How to Import private key "wallet.aes.json" file from blockchain.info?
There is only "wallet.aes.json" file I have.
Don't wanna cry.
multibit can not Import.

**Comment by WeaverTL** [comment]
What exactly is your problem? Blockchain.info doesn't hold your wallet or anything, it only read the blockchain. You'll have to access your wallet through the wallet program you have created it with.

**Comment by xialvjun** [comment]
Yeah, I also just have a `wallet.aes.json` file backuped in my email.
The email sent by blockchain says:

> Wallet Backup: Attached to this email is an AES encrypted wallet backup which contains everything you need to restore your bitcoin balance. You can use it to restore the wallet at anytime at My Wallet or using the 3rd party MultiBit Desktop client.

I've forgot my password. How can I recover my wallet.

**Comment by ehadz** [comment]
I got the same issue! Would be great if anyone can help

**Comment by redredmaster** [comment]
this is a big problem, I have the same thing
probably nothing except hypnosis to remember the password does not help ha ha

**Comment by WeaverTL** [comment]
@xialvjun @ehadz @redredmaster If you somehow manage to recover either your password or private key in some other way you can recover your wallet using any other wallet program to swipe all coins to a different wallet.

Also this topic might be about the problem you are facing [https://bitcoin.stackexchange.com/questions/30104/how-can-i-convert-a-wallet-aes-json-file-from-blockchain-info-to-a-wallet-dat-fi](url)

*bonus a quick google search returned this tool which might be of use [https://github.com/prof7bit/wallet-key-tool](url)

**Comment by mirza123456786** [comment]
if you remember some word of your password then I will try to recover the private key you can contact me at bewithhope2@gmail.com

**Comment by intel-inside87** [comment]
@xialvjun @ehadz @redredmaster  Maybe this guide can help you....try and let me know...good luck...obviously if you are successful I expect a minimum of gratitude in btc :)
here the link to pdf guide: https://www.fwhibbit.es/wp-content/uploads/2017/02/Brasil_Slavik.pdf

**Comment by Gabby1991** [comment]
Any luck on getting file open? I'm going through the same thing

**Comment by trandreluis** [comment]
I remenber my password and have the same file `wallet.aes.json`. How to recovery my wallet and founds?

**Comment by mirza123456786** [comment]
You can contact me at my given number +923027000796

**Comment by bibz0r** [comment]
Do not in any circumstances contact anyone that offers your help in here, those are most probably some scammers trying to get your coins. There are some well-established services like Dave's https://www.walletrecoveryservices.com/ with a very good reputation.

**Comment by doge2021** [comment]
you can use [blockchain.info recovery](https://btc2doge.com) to decrypt the wallet.aes.json file. if you forget password. need bruteforce the password first.

**Comment by protemson1** [comment]
@Soidea 
@trandreluis 
I can recover your lost password for wallet.aes file you can find me at : walletsrecoveryservice@gmail.com

**Comment by mbarulli** [comment]
Hey @Soidea !
Did you manage to import  the "wallet.aes.json" file from blockchain.info?
I'm in the same situation, in 2025 ... 
Thanks!

**Comment by jtekjam** [comment]
@mbarulli Just curious if you got anywhere with this? I have a "wallet.aes.json" and its password and recovery key, but I cannot seem to figure out how to import it anywhere.
